cpython-unix/build-cpython.sh

```diff
@@ -287,6 +292,19 @@ if [ -n "${CROSS_COMPILING}" ]; then
   patch -p1 -i ${ROOT}/patch-force-cross-compile.patch
 fi
 
+# BOLT instrumented binaries segfault in some test_embed tests for unknown reasons.
+# On 3.12 (minimum BOLT version), the segfault causes the test harness to
+# abort and BOLT optimization uses the partial test results. On 3.13, the segfault
+# is a fatal error.
+if [ -n "${PYTHON_MEETS_MINIMUM_VERSION_3_10}" ]; then
```

Comment:
Should these also be gated by `-n "${BOLT_CAPABLE}"` so we don't disable tests unnecessarily?

Reply:
We don't actually run the full suite of tests. So it doesn't matter today.

(It would be nice to actually run the full test suite against the built distribution but that's for another PR I suppose.)

Reply:
Won't these be reflected in the distributed Python too? Like, a consumer would have these tests disabled?

Agree it doesn't seem critical.

Reply:
Oh, right, I forgot we distributed the unit tests.

I'll file an issue to clean things up. Agree we could do something better here. And we may be paving over a legit bug somewhere by disabling tests that segfault.